docs/source/topics/proc_accessing-the-openshift-cluster-with-oc.adoc

```diff
@@ -50,8 +50,8 @@ $ oc get co
 {prod} disables the Cluster Monitoring Operator by default.
 ====
 
-See link:{crc-gsg-url}#troubleshooting-codeready-containers_gsg[Troubleshooting {prod}] if you cannot access the {prod} OpenShift cluster.
+See link:{crc-gsg-url}#troubleshooting-codeready-containers_gsg[Troubleshooting {prod}] if you cannot access the {prod} cluster.
```

Comment:
Just `{prod}`, no `{ocp}` or `{openshift}`?

Reply:
Good point. Perhaps we should say "The {ocp} cluster managed by {prod}".